src/common/common.go

```diff
@@ -248,35 +251,84 @@ func parseOpenshiftVersionIntoMajorMinorZOnly(version string) string {
 }
 
 // HasValidvSphereCredentials returns true if the the platform is
-// vSphere and the install config overrides contains real
+// vSphere and the install config or equivalent overrides contains real
 // credential values and not placeholder values.
 // Deprecated credential fields are not considered valid.
-func HasValidvSphereCredentials(ctx context.Context, ic inventory_client.InventoryClient, log logrus.FieldLogger) bool {
+func HasValidvSphereCredentials(ctx context.Context, ic inventory_client.InventoryClient, kc k8s_client.K8SClient, log logrus.FieldLogger) bool {
 	cluster, callErr := ic.GetCluster(ctx, false)
+	if cluster != nil {
```

Comment:
maybe lets add comment here that describes why it is ok not to have cluster?

Reply:
Not relevant to this pr but i wonder why we could create some secret with those credentials as manifest and reference it to controller?

Reply:
(but I also understand that it may require additional work, so don't want to block this PR)